cloud/amazon/ec2_vpc_peering.py

```diff
@@ -0,0 +1,596 @@
+            - Required for all states except "list"
+        required: false
+        default: null
+    peer_vpc:
+        description:
+            - VPC ID, CIDR block or name tag of the accepting VPC
+            - Required for all states except "list"
+        required: false
+        default: null
+        aliases: ['dest_vpc']
+    peer_owner_id:
+        description:
+            - Account ID of the owner of the accepting VPC, if different.
+        required: false
+        default: null
+    state:
+        description:
+            - Sets desired state for this VPC peering connection. If set to 'pending', if the connection does not exist, it will be created in the 'pending-acceptance' state. If set to 'present' or 'active', and both the source and dest VPC are owned by you, this module will create and set the connection to be active.
+        required: false
+        default: 'present'
+        choices: ['absent', 'present', 'active', 'pending', 'list']
+    force_accept:
+        description:
+            - Normally, this module only accepts pending connections that it owns.
+            - Enabling this option attempts to accept all connections seens that are pending acceptance.
+        required: false
+        default: false
+    ignore_rejection:
+        description:
+            - By default, this module does not attempt to re-create a connection that has been rejected.
+            - Enabling this option will create connections even if they have been rejected.
+        required: false
+        default: false
+    always_delete:
+        description:
+            - By default, when removing connections, connections in a pending-acceptance state will be rejected.
+            - Enabling this flag will always delete a connection whether it is pending acceptance or not.
+        required: false
+        default: false
+
+extends_documentation_fragment: aws
+'''
+
+EXAMPLES = '''
+# Create a VPC peering connection between ovpc1 and ovpc2
+- local_action: 
+    module: ec2_vpc_peering
+    name: 'ovpc1-to-ovpc2'
+    source_vpc: 'ovpc1'
+    dest_vpc: 'ovpc2'
+    region: 'us-west-1'
+    state: 'present'
+    
+# List all VPC peering connections
+- local_action:
+    module: ec2_vpc_peering
+    region: 'us-east-1'
+    state:  'list'
+  register: peering_list
+
+- debug: var=peering_list
+
+# Create a VPC peering connection from your VPC with CIDR 10.0.0.0/16 
+# to vpc ID vpc-01234567, owned by the account ID 012345678901
+- local_action: 
+    module: ec2_vpc_peering
+    source_vpc: '10.0.0.0/16'
+    dest_vpc: 'vpc-01234567'
+    peer_owner_id: '012345678901'
+    region: 'us-east-1'
+    state: 'present'
+
+# Ensure any peering connections between the VPCs with the CIDRS 10.10.1.0/24
+# and 10.10.2.0/24 are not present.
+- local_action: 
+  module: ec2_vpc_peering
+  source_vpc: '10.10.1.0/24'
+  dest_vpc: '10.10.2.0/24'
+  region: 'us-west-1'
+  state: 'absent'
+'''
+
+from boto.exception import EC2ResponseError
```

Comment:
look at other amazon modules for example, boto should be imported in a exception handler, set a var to return a 'nice' message to user that the requirement is missing if not found.